packages/magento-customer/components/ChangePasswordForm/ChangePasswordForm.tsx

```diff
@@ -22,49 +22,53 @@ export function ChangePasswordForm() {
     ChangePasswordMutation,
     ChangePasswordMutationVariables & { confirmPassword?: string }
   >(ChangePasswordDocument)
-  const { muiRegister, handleSubmit, required, watch, data, formState, error } = form
+  const { handleSubmit, required, data, formState, error, control } = form
+  const [remainingError, inputError] = graphqlErrorByCategory({
+    category: 'graphql-input',
+    error,
+  })
+
   const submitHandler = handleSubmit(() => {})
-  const pass = watch('newPassword')
 
   return (
     <Form onSubmit={submitHandler} noValidate>
       <FormRow>
-        <TextField
+        <PasswordElement
+          control={control}
+          name='currentPassword'
```

Comment:
The current password should not have any validation as this will break when future password requirements change. For example when the future requirement is 12 characters and 4 groups, you wont be able to enter your current password.